app/models/resource.rb

```diff
@@ -59,7 +59,10 @@ class Resource < ApplicationRecord
   scope :category_names, ->(names) { tag_names(:categories, names) }
   scope :sector_names,   ->(names) { tag_names(:sectors, names) }
   scope :featured, -> (featured=nil) { featured.present? ? where(featured: featured) : where(featured: true) }
-  scope :kind, -> (kind) { where("kind like ?", kind ) }
+  scope :kind, ->(kinds) {
```

Comment:
this is so much better!

you didn't want to rename the scope to `kinds` instead of `kind`? i've been going back and forth on whether or not i think they should be singular or plural when sending in plural values.